runner.py

```diff
@@ -60,7 +61,9 @@
         'results': []
     }
     is_successful = True
-
+    if os.path.isdir(args.configs):
```

Comment:
Shouldn't this code be recursive? What if we have configs in a folder, which is in a folder itself? This behaviour doesn't seem quite obvious to me